x-pack/plugins/infra/public/pages/metrics/inventory_view/components/waffle/conditional_tooltip.tsx

```diff
@@ -91,10 +99,18 @@ export const ConditionalToolTip = withTheme(
         {metrics.map((metric) => {
           const metricName = SnapshotMetricTypeRT.is(metric.name) ? metric.name : 'custom';
           const name = SNAPSHOT_METRIC_TRANSLATIONS[metricName] || metricName;
-          const formatter = createInventoryMetricFormatter({ type: metricName });
+          // if custom metric, find field and label from waffleOptionsContext result
+          // because useSnapshot does not return it
+          const customMetric =
+            name === 'custom' ? customMetrics.find((item) => item.id === metric.name) : null;
```

Comment:
For each custom metric, each custom metric is searched to find the matching ID.  I'm assuming here there will not be many custom metrics.